docs/widgetbook-cloud/review.mdx

```diff
@@ -18,14 +18,6 @@ To use Widgetbook Cloud _Review_ your setup has to meet the following requiremen
 - Use the `widgetbook-cli` from the [widgetbook_cli](https://pub.dev/packages/widgetbook_cli) pub.dev package.
 - Setup a GitHub webhook
 
-## Setup the CLI
-
-For instructions on how to setup the CLI see [our docs](/widgetbook-cli/overview) or just run:
-
-```bash
-dart pub global activate widgetbook_cli
-```
-
 ### CLI Example (GitHub)
 
 The CLI extracts information from `widgetbook_generator` and also obtains branch information from git.
```

Comment:
I also wanted to surface that in the webview https://docs.widgetbook.io/widgetbook-cloud/review the `uses:` show up as:
<img width="335" alt="Screen Shot 2022-07-25 at 9 15 58 PM" src="https://user-images.githubusercontent.com/10517945/180902446-b026342e-273c-4144-858e-29b1ff5cb3a4.png">

It might be good to either unhide them or make a note to users that they need to be replaced

Reply:
Yeah, that's a problem from the docs page. I'll create an issue that I link in the docs for now until they fix the problem.